### ISSUE TITLE
https://sslvpn.club blocked

### ISSUE THREAD

**Issue by easella** [davidebangelista/Goguardian-bypasser#3]
![image](https://user-images.githubusercontent.com/74786054/115121679-08f87e80-9f82-11eb-9b87-436d9286b8ec.png)

By the way, iframes do not bypass my schools blocking system, which is contentkeeper

**Comment by davidebangelista** [comment]
Try these on the VOX browser. I'm currently trying newer proxies as old ones die:
(A) https://holyubofficial.net/?w
(B)https://interrupttransparent.live/?a 
(C) https://cereal.gq/?a 
(D) https://graduate.cf/?w
Please tell me which ones work, I need to edit proxies for each school district.

**Comment by davidebangelista** [comment]
If you have Reddit, send me which ones work as I don't check here very often.
https://www.reddit.com/user/TOR-anon1/ or u/TOR-anon1

**Comment by jhon-cena77** [comment]
Yes my school also has ContentKeeper block i-frames, most proxies are suspicious and sketch you can try Google searching "pro-xy sites", most work, but loading the site through proxy doesn't work, just says ERR_NO_NAME or something.

Any new information?
